superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/DefaultValue.tsx

```diff
@@ -62,7 +62,7 @@ const DefaultValue: FC<DefaultValueProps> = ({
   ) : (
     <SuperChart
       height={25}
-      width={250}
+      width={180}
```

Comment:
This feels slightly tight:
![image](https://user-images.githubusercontent.com/33317356/120186412-25900380-c21c-11eb-8fce-e50c3d00ba51.png)
Maybe we should leave this change to a separate PR if it's needed so we can concentrate on the design aspects there and make this PR focus only on functional fixes related to the first value option?

Reply:
I did it because of in Native filters modal it's too large, but I can cancel it here